client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -80,12 +81,21 @@ class PluginUpload extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const {
+			translate,
+			upgradeJetpack,
+			siteId,
+		} = this.props;
 
 		return (
 			<Main>
 				<HeaderCake onClick={ this.back }>{ translate( 'Upload plugin' ) }</HeaderCake>
-				{ this.renderUploadCard() }
+				{ upgradeJetpack && <JetpackManageErrorPage
```

Comment:
Yeah I think the ternary would normally be the right choice here, except we have #16759 coming which will add another 'branch' and another condition for rendering the upload card.